java-test-samples/schema-and-contract-testing/src/test/java/com/example/SchemaTest.java

```diff
@@ -0,0 +1,100 @@
+package com.example;
+
+import java.io.IOException;
+import java.util.Set;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.ValidationMessage;
+
+/**
+ * Unit test for simple App.
+ */
+public class SchemaTest extends BaseJsonSchemaValidatorTest
+{
```

Comment:
Better to avoid inconsistency in code formats. Use `.editorconfig` and format code just before pushing.